openlibrary/plugins/openlibrary/js/readmore.js

```diff
@@ -0,0 +1,28 @@
+export function initReadMoreButton() {
+    var el, p, up, height;
```

Comment:
`const` these and toss them inside each of the functions. They don't need to be globals.